blog/releases/announcing-knative-v0-23-release.md

```diff
@@ -0,0 +1,202 @@
+
+
+#### RabbitMQ Eventing v0.23
+
+<!-- Original notes are here: https://github.com/knative-sandbox/eventing-rabbitmq/releases/tag/v0.23.0 -->
+
+#### 🚨 Breaking or Notable Changes
+
+- requires [messaging-topology-operator](https://github.com/rabbitmq/messaging-topology-operator) version `v0.8.0` to be installed as well as [cluster-operator](https://github.com/rabbitmq/cluster-operator) version `v1.6.0`
+
+#### 🐞 Bug Fixes
+
+- Conformance tests for Broker now work
+- If a Broker / Trigger is deleted but there's no secret to talk to the rabbit cluster, we now remove the resources and log that rabbitmq resources might have been leaked. This happens only when not using the Rabbit Cluster Operator. This left undeletable resources, so this change seems more user friendly. [#271](https://github.com/knative-sandbox/eventing-rabbitmq/pull/271)
+
+### Client v0.23
+
+<!-- Original notes are here: https://github.com/knative/client/blob/main/CHANGELOG.adoc#v0230-2021-05-18 -->
+
+#### 💫 New Features & Changes
+
+- Update Eventing Sources APIServerSource, ContainerSource, SinkBinding API to v1. [#1299](https://github.com/knative/client/pull/1299)
+- Update Eventing Source PingSource API to v1beta2. [#1299](https://github.com/knative/client/pull/1299)
+- Add number of instances to describe command. [#1289](https://github.com/knative/client/pull/1289)
+
+#### 🐞 Bug Fixes
+
+- Fix for serviceaccounts "default" not found flaky issue. [#1312](https://github.com/knative/client/pull/1312)
+- Fix number of instances *int32 type in describe commands. [#1312](https://github.com/knative/client/pull/1312)
+- Use fully qualified test images. [#1307](https://github.com/knative/client/pull/1307)
+- Fix documentation of the configuration options. [#1297](https://github.com/knative/client/pull/1297)
+- Respect -o in list commands in case if no data present. [#1276](https://github.com/knative/client/pull/1276)
+
+
+### Operator v0.23
+
+<!-- Original notes are here: https://github.com/knative/operator/releases/tag/v0.23.0 -->
+
+#### 🐞 Bug Fixes
+
+- Apply high availability settings to kourier as well. [#579](https://github.com/knative/operator/pull/579)
+- Clear OwnerReferences on knative-local-gateway Service. [#567](https://github.com/knative/operator/pull/567)
+- Set the namespace of the service knative-local-gateway to the istio namespace. [#590](https://github.com/knative/operator/pull/590)
+- Add pingsource-mt-adapter into HA list. [#591](https://github.com/knative/operator/pull/591)
+- Return an actual newest version of the directory latest is not available. [#576](https://github.com/knative/operator/pull/576)
+- Allow for all Ingresses to be disabled. [#571](https://github.com/knative/operator/pull/571)
+
+#### 🧹 Clean up
+
+- Simplify istio installation for e2e test. [#564](https://github.com/knative/operator/pull/564)
+- Use deployment namespace to determine gateway namespace for Kourier. [#577](https://github.com/knative/operator/pull/577)
```

Comment:
```suggestion
- Use the `deployment` namespace to determine gateway namespace for Kourier. [#577](https://github.com/knative/operator/pull/577)
```
Is it actually called `deployment`?

Reply:
Don't think is a namespace named `deployment`
Looking at the code in the [PR 577](https://github.com/knative/operator/pull/577/files#diff-b4c139ccd3f76f7257855c955ca2d3bceaf5786788d965f88fbcfc34a2d3585dR46)  is setting an environment variable to the namespace value where the kourier gateway deployment is deploy.